magnetic/Glk/glk.c

```diff
@@ -5877,6 +5924,15 @@ gms_startup_code (int argc, char *argv[])
     {
       gms_gamefile = argv[argv_index];
       gms_game_message = NULL;
+#ifdef GARGLK
+    {
+      char *s;
+      s = strrchr(gms_gamefile, '\\');
+      if (s) garglk_set_story_name(s+1);
+      s = strrchr(gms_gamefile, '/');
+      if (s) garglk_set_story_name(s+1);
+    }
+#endif
```

Comment:
This is incorrectly indented: it should be moved one character over to the right.

Reply:
Is this the correct indentation?
```
   */
  if (argv_index == argc - 1)
    {
      gms_gamefile = argv[argv_index];
      gms_game_message = NULL;
#ifdef garglk
      {
        char *s;
        s = strrchr(gms_gamefile, '\\');
        if (s) garglk_set_story_name(s+1);
        s = strrchr(gms_gamefile, '/');
        if (s) garglk_set_story_name(s+1);
      }
#endif
    }
  else
```

Reply:
Yes... although since these interpreters will be requiring C11, or at least C99, the braces are actually unnecessary.  So I really think the braces should be removed, and have the Gargoyle-specific code lined up with the existing code (i.e. 6 spaces).

Sorry for not noticing that earlier.